packages/api/src/controllers/access-control.ts

```diff
@@ -138,9 +138,14 @@ app.post(
     if (
       playbackPolicyType !== "public" &&
       req.body.pub === req.config.accessControlAdminPubkey &&
+      req.config.accessControlAdminPubkey !== "" &&
+      req.config.accessControlAdminPubkey &&
       req.body.pub !== "" &&
       req.body.pub
     ) {
+      console.log(
+        `access-control: gate: allowing admin access for contentId=${content.id} playbackId=${playbackId}, user=${user.id} is admin, pubKey=${req.config.accessControlAdminPubkey} is adminPubKey`
```

Comment:
Hmm, this part is weird: `user=${user.id} is admin`

Shouldn't you actually log `user.admin` here? I don't think there's a guarantee on `user` being an admin here (in fact its probably not, since it's the owner of the video, not the caller)